### DISCUSSION TITLE
My install keeps trying to download Symbols and failing

### DISCUSSION THREAD

**Discussion by FireFrenzy** [valinet/ExplorerPatcher#2685] [Q&A] [answered]
I've turned off my firewall in case that was blocking it, which i did because it came back with a message about "being connected to the internet or the server on the far end being down" or something to that effect. 

I've fully reinstalled manually. 

Like this is clearly not the biggest deal in the universe but its annoying enough i have found and unlocked my old github account and probably already broken every single piece of github etiquette there is.

**Comment by pyrates999** [comment, marked as answer]
Please keep trying until symbols are available to download for where you are.

**Reply by FireFrenzy** [reply]
Not the answer i was hoping for but not an unexpected one... Is there even the roughest of guesstimates about where The Netherlands are in the queue?

**Reply by pyrates999** [reply]
Up to MS.

**Reply by FireFrenzy** [reply]
Figured as much! Thanks!

**Reply by pyrates999** [reply]
You're welcome.
